src/sisl/geom/neighs/_neigh_operations.py

```diff
@@ -306,12 +310,12 @@ def get_all_unique_pairs(
 
     N_ats = list_array.shape[0]
 
-    i_pair: cython.int = 0
+    i_pair: cython.size_t = 0
 
     ref_xyz: cnp.float64_t[:] = np.zeros(3, dtype=np.float64)
     neigh_isc: cnp.int64_t[:] = np.zeros(3, dtype=np.int64)
 
-    neighs: cnp.int64_t[:, :] = np.zeros((max_npairs, 5), dtype=np.int64)
+    neighs: cnp.int64_t[:, :] = np.zeros([max_npairs, 5], dtype=np.int64)
```

Comment:
Why do you prefer this? I thought creating a list was slower and used more memory. I always use tuples for the shapes of arrays.

Reply:
For me the clarity of reading `([])` is much better compared to `(())`, I know this is a personal preference. The overhead of the list vs. tuple is really negligible in these instantiation processes, the `np.zeros|whatever` is by far more heavy (I would assume).